manager/orchestrator/replicated/update_test.go

```diff
@@ -204,6 +211,12 @@ func testUpdaterRollback(t *testing.T, rollbackFailureAction api.UpdateConfig_Fa
 	assert.Equal(t, observedTask.Status.State, api.TaskStateNew)
 	assert.Equal(t, observedTask.Spec.GetContainer().Image, "image1")
 
+	if !setMonitor {
```

Comment:
Rather than just return, is there some other way we can set a test for this regression?  Possibly changing the default value, so that we can set a timeout and hence detect if this regresses again?